lib/internal/Magento/Framework/Webapi/ServiceInputProcessor.php

```diff
@@ -219,10 +219,6 @@ protected function _createFromArray($className, $data)
 
         // Secondary method: fallback to setter methods
         foreach ($data as $propertyName => $value) {
-            if (isset($constructorArgs[$propertyName])) {
-                continue;
-            }
-
             // Converts snake_case to uppercase CamelCase to help form getter/setter method names
```

Comment:
this looks wrong, as it eliminates main goal of the fix, introduced by @phoenix128